openstack/orchestration/v1/stacks/requests.go

```diff
@@ -197,12 +181,12 @@ func (opts AdoptOpts) ToStackAdoptMap() (map[string]interface{}, error) {
 		s["parameters"] = opts.Parameters
 	}
 
-	if opts.Timeout == 0 {
-		return nil, errors.New("Required field 'Timeout' not provided.")
+	if opts.Timeout != 0 {
+		s["timeout"] = opts.Timeout
 	}
 	s["timeout_mins"] = opts.Timeout
 
-	return map[string]interface{}{"stack": s}, nil
```

Comment:
Maybe they changed the request structure sometimes soon? Making a request to the api will not work unless this change is made